webapp/views/wpt-results.js

```diff
@@ -580,6 +585,7 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
   connectedCallback() {
     super.connectedCallback();
     this.addEventListener('triagemetadata', this.reloadPendingMetadata);
+    window.addEventListener('getSubtestRows', this.handleGetSubtestRows.bind(this));
```

Comment:
```suggestion
    this.addEventListener('getSubtestRows', this.handleGetSubtestRows.bind(this));
```
We should bind it to the closest parent DOM

Reply:
`getSubtestRows` should be changed according per comment above

Reply:
done 👍